Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Display/XamlRenderer.cs

```diff
@@ -999,24 +1005,33 @@ private void RenderItalicRun(InlineCollection inlineCollection, ItalicTextInline
         /// <param name="context"> Persistent state. </param>
         private void RenderStrikethroughRun(InlineCollection inlineCollection, StrikethroughTextInline element, RenderContext context)
         {
-            Span span = new Span
+            Span span = new Span();
+
+            if (TextDecorationsSupported)
             {
-                FontFamily = new FontFamily("Consolas")
-            };
+                span.TextDecorations = TextDecorations.Strikethrough;
+            }
+            else
+            {
+                span.FontFamily = new FontFamily("Consolas");
+            }
 
             // Render the children into the inline.
             RenderInlineChildren(span.Inlines, element.Inlines, span, context);
 
             AlterChildRuns(span, (parentSpan, run) =>
```

Comment:
What about handling (not saying it does handle it, but it should) ~~**strikethrough bold text**~~? Or ~~*strikethrough italic text*~~? If we handled that properly wouldn't it be handled in `AlterChildRuns`? 

NOTE: I have no clue on the code-flow of this